lib/intercom-rails/script_tag.rb

```diff
@@ -18,6 +18,7 @@ def initialize(options = {})
       self.controller = options[:controller]
       @show_everywhere = options[:show_everywhere]
       self.user_details = options[:find_current_user_details] ? find_current_user_details : options[:user_details]
+      remove_user_cookie_on_logout if http_request?
```

Comment:
Tests on scriptTag Class fails without http_request? 
If there is no option[:controller] (I can't find any real life use case were it happens though but it does in unit testing) then initialise shouldn't fail just because we want to remove cookies on non existant controller